src/common/screens/ImageGallery/ImageGallery.jsx

```diff
@@ -62,24 +64,24 @@ const ImagesGalleryScreen = ({
           endYear={endYear}
         />
       </div>
-      <ImageGallery sourceImage={image} />
+      <ImageGallery sourceImage={images} />
       <Pagination
         setCurrentPage={setCurrentPage}
-        totalPage={totalPage(total_hits)}
+        totalPage={totalItems}
         pageCurrent={pageIndex}
       />
     </div>
   )
 }
 
 const mapStateToProps = state => ({
-  images: state.imagesGalleryPage.items,
-  isLoading: state.imagesGalleryPage.isLoading,
-  searchText: state.imagesGalleryPage.searchText,
-  pageIndex: state.imagesGalleryPage.pageIndex,
+  images: getImagesSelector(state),
+  isLoading: getIsLoading(state),
+  searchText: getSearchText(state),
+  pageIndex: getCurrentPage(state),
   startYear: getStartYear(state),
   endYear: getEndYear(state),
-  totalItems: state.imagesGalleryPage.totalItems,
+  totalItems: getTotalItemsOnPageSelector(state),
 })
```

Comment:
there is another nicer selector in `reselect` package called: `createStructuredSelector` which accepts an object, instead if this:
```
const mapStateToProps = (state, props) => ({
  images: getImagesSelector(state, props),
  isLoading: getIsLoading(state, props),
  searchText: getSearchText(state, props),
  pageIndex: getCurrentPage(state, props),
}) 
```
you can prefer this:
```
import { createStructuredSelector } from 'reselect'

const mapStateToProps = createStructuredSelector({
  images: getImagesSelector,
  isLoading: getIsLoading,
  searchText: getSearchText,
  pageIndex: getCurrentPage,
}) 
```

The second is better because you dont need to pass **state** and **props** every time you use a selector :)